libpod/networking_linux.go

```diff
@@ -225,21 +226,55 @@ func (r *Runtime) setupRootlessNetNS(ctr *Container) error {
 
 	isSlirpHostForward := false
 	disableHostLoopback := true
+	var outboundAddr string
+	outboundAddr = nil
```

Comment:
The default string type of golang is null, which is "", can not use nil in string